app/views/communities/_banusers.html.erb

```diff
@@ -0,0 +1,68 @@
+<%# function DoSubmit(){
+  ,onsubmit="DoSubmit();"
+  document.is_banned.value = true;
+  return true;
+} %>
+
+
+<div class="col-md-4 offset-md-4">  
+   <button type="button" class="btn btn-outline-primary" data-toggle="modal" data-target="#myModal"> Ban Users </button>
+  </div>
+  <div class="col-md-4 offset-md-4">  
+    <div class="card">
+      <div class="card-body">
+        This is some text within a card body.
+      </div>
+    </div>
+  </div> 
+</div>                                                
+
+
+<div class="modal fade" id="myModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+     <div class="modal-header">
+        <h5 class="modal-title">Ban a User:</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+
+      <div class="modal-body">
+        <%= form_for @banneduser do |form| %> 
+          <div class="form-group">
+           <%# <%= label_tag :Community_id %>  
+            <%= form.hidden_field :community_id, :value => params[:id], class: "form-control" %> 
+           </div> 
+
+          <div class="form-group">
+           <%= label_tag :Enter_User_Name %>
+            <%# <div id= "search-data" class="form-search search-data" data-skills="<%= @username.to_json %>
+            <%= form.text_field :username, autocomplete: :off, class: "form-control search-input", placeholder: "Search by username" %>
+            <%# <%= form_tag community_path, method: :get do %>
```

Comment:
Remove comments if they are not required

Reply:
Comments are Removed